docs/tools/cli/service/kafka-acl.md

```diff
@@ -0,0 +1,71 @@
+---
+title: avn service kafka-acl
+---
+
+Full list of commands for `avn service kafka-acl`.
```

Comment:
Should we consider consolidating Aiven ACLs commands and Kafka-native ACLs commands into a single topic?